src/plugins/unified_search/public/filter_bar/filter_view/index.tsx

```diff
@@ -36,13 +38,20 @@ export const FilterView: FC<Props> = ({
 }: Props) => {
   const [ref, innerText] = useInnerText();
 
-  let title =
-    errorMessage ||
-    i18n.translate('unifiedSearch.filter.filterBar.moreFilterActionsMessage', {
-      defaultMessage: 'Filter: {innerText}. Select for more filter actions.',
-      values: { innerText },
-    });
+  let filterString = i18n.translate('unifiedSearch.filter.filterBar.filterString', {
+    defaultMessage: 'Filter: {innerText}.',
+    values: { innerText },
+  });
+  if (!readOnly) {
+    filterString = `${filterString} ${i18n.translate(
+      'unifiedSearch.filter.filterBar.filterActionsMessage',
+      {
+        defaultMessage: 'Select for more filter actions.',
+      }
+    )}`;
+  }
```

Comment:
Concatenating translated strings doesn't always work the way we hope, because some locales will want things switched around in a way this would prevent.

I recommend:
```
  const filterString = readOnly
    ? i18n.translate('unifiedSearch.filter.filterBar.filterString', {
        defaultMessage: 'Filter: {innerText}.',
        values: { innerText },
      })
    : i18n.translate('unifiedSearch.filter.filterBar.filterActionsMessage', {
        defaultMessage: 'Filter: {innerText}. Select for more filter actions.',
        values: { innerText },
      });
```

Reply:
That makes total sense - that's honestly a much cleaner way of doing it anyway, thanks! 👍